dev-packages/browser-integration-tests/suites/public-api/instrumentation/onError/non-string-arg/test.ts

```diff
@@ -0,0 +1,24 @@
+import { expect } from '@playwright/test';
+import type { Event } from '@sentry/types';
+
+import { sentryTest } from '../../../../../utils/fixtures';
+import { getFirstSentryEnvelopeRequest } from '../../../../../utils/helpers';
+
+sentryTest('should onerror calls with non-string first argument gracefully', async ({ getLocalTestPath, page }) => {
```

Comment:
I think something's missing here 😅 

```suggestion
sentryTest('should handle onerror calls with non-string first argument gracefully', async ({ getLocalTestPath, page }) => {
```

Reply:
or
```suggestion
sentryTest('should catch onerror calls with non-string first argument gracefully', async ({ getLocalTestPath, page }) => {
```